gputop/remote/gputop-ui.js

```diff
@@ -255,17 +224,6 @@ GputopUI.prototype.update_counter = function(counter) {
         text_value *= 1000000;
     }
```

Comment:
I think we want to also make sure we convert mhz units to hz or us units to ns like this when plotting data in the trace view graphs.

If we introduced a `counter.units_scale` constant (default 1) then in `read_counter_xml` when we initialize `counter.duration_dependant` we could also check if (units === "mhz") { units = "hz"; counter.units_scale = 1000;} and if (units === "us") { units = "ns"; counter.units_scale = 1000; }

This in `append_counter_data` we can unconditionally do d_value *= counter.units_scale and max *= counter.units_scale.